source/documentation/upgrade_guide/topics/installing_virtual_product_hypervisors_from_local_repository.adoc

```diff
@@ -0,0 +1,88 @@
+[id="installing_virtual_product_hypervisors_from_local_repository_{context}"]
+= Installing {virt-product-shortname} hypervisors from a local repository
+
+If your system uses a private {virt-product-fullname} ({virt-product-shortname}) environment, but without Red Hat Satellite, you might need to install {virt-product-shortname} hypervisors (RHV-H) from a repository hosted on a local RHEL system instead of the Red Hat hosted Content Delivery Network (CDN).
+
+.Procedure
+. On the system hosting the offline repository, create a file `/etc/yum/repos.d/rhvh-mirror.repo` similar to:
```

Comment:
Use complete sentence for lead-in sentence for a list. See IBM Style Guide: https://www.ibm.com/docs/en/ibm-style?topic=format-lists#lead-in-wording